pymc3/distributions/posterior_predictive.py

```diff
@@ -42,7 +42,7 @@
 )
 from ..exceptions import IncorrectArgumentsError
 from ..vartypes import theano_constant
-from ..util import dataset_to_point_dict, chains_and_samples, get_var_name
+from ..util import dataset_to_point_list, chains_and_samples, get_var_name
```

Comment:
Is this change related to the windows bug? I'm still fine with it being here, it's just for my understanding :smile:

Reply:
no, unrelated.. I just noticed the `FIXME` comment and wanted to get a DeprecationWarning in before 3.10